ush/ufsda/archive.py

```diff
@@ -21,15 +22,15 @@ def atm_diags(config):
     r2d2_config = NiceDict(r2d2_config)
     # get list of obs to process and their output files
     obs_list_yaml = config['OBS_LIST']
-    obs_list_config = Configuration(obs_list_yaml)
-    obs_list_config = ufsda.yamltools.iter_config(config, obs_list_config)
+    obs_list_config = YAMLFile(path=obs_list_yaml)
     for ob in obs_list_config['observers']:
         # first get obs
         r2d2_config.pop('file_type', None)
         r2d2_config['type'] = 'ob'
         r2d2_config['provider'] = config['provider']
         r2d2_config['start'] = config['window_begin']
         r2d2_config['end'] = r2d2_config['start']
+        print(ob)
```

Comment:
this was my 'oops' that should be removed

Reply:
`print(ob)` removed.   I didn't even notice this.

Reply:
Change committed at 23ef107